cloud/jenkins/pgo-v2-operator-aws-openshift-latest.yml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: pgo-v2-operator-aws-openshift-latest
+    project-type: pipeline
+    description: |
+        Do not edit this job through the web!
+    triggers:
+      - timed: "0 6 * * 3"
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/Percona-Lab/jenkins-pipelines.git
+            branches:
+            - 'master'
+            wipe-workspace: false
+      lightweight-checkout: true
+      script-path: cloud/jenkins/pgo_v2_operator_aws_openshift-latest.groovy
+    parameters:
```

Comment:
I would recommend to remove parameters from here because we are bad at syncing them with the pipeline code. So better to just have them in pipeline code.